example/deployment-aws.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       containers:
       - name: etcd-operator
         image: quay.io/coreos/etcd-operator
```

Comment:
@djMax change this to quay.io/coreos/etcd-operator:v0.0.2?

we will update this deployment example when we bump etcd operator version.